icons/telescope.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M8.5 14.7 6 11" />
+  <path d="M17.4 10.1 14 4.9" />
+  <path d="M19 2.4c-.4-.4-1-.5-1.4-.2L2 14l2 3 17.5-9c.5-.2.6-.8.3-1.2Z" />
+  <path d="M13 22v-9.6" />
+  <path d="m9 21 4-4 4 4" />
```

Comment:
The curvature is noticeably off:
![image](https://github.com/lucide-icons/lucide/assets/17746067/4cb033e0-7826-49c9-917d-4fa11006fb30)

I also think we ought to add ~1px rounding to the base (and compensate for its reduced width):
![image](https://github.com/lucide-icons/lucide/assets/17746067/f93fe9dc-b071-49c4-ba66-67ebe785df59)

```suggestion
  <path d="m5.55 10.3 2.89 4.78" />
  <path d="m13.89 4.68 3.41 5.64" />
  <path d="m20.12 2.96a2 2 0 0 0-2.83-.62L2.44 12.37a1 1 0 0 0-.3 1.34l1.67 2.8a1 1 0 0 0 1.34.37l15.8-8.5a2 2 0 0 0 .76-2.79Z" />
  <path d="M12 22v-8.8" />
  <path d="m8 21 4-4 4 4" />
```